EcoSoapBankTests/UtilityTests.swift

```diff
@@ -28,4 +28,16 @@ class UtilityTests: XCTestCase {
         }
         XCTAssertEqual(object.int, 42)
     }
+
+    func testDate() {
```

Comment:
Maybe rename this to be a bit more specific. I had to look in UtilityFunctions to realize that it was a custom init that you were testing.

``` swift
func testCustomDateInitializer()
```

Reply:
Good call. Done.